packages/backend/src/App.ts

```diff
@@ -225,8 +225,8 @@ export default class App {
             App.initNodeProcessMonitor();
         }
 
-        const expressApp = await this.initExpress();
-        this.initSentry(expressApp);
+        await this.initExpress();
```

Comment:
happy to have this instead: 
```ts
const expressApp = express();
await this.initSentry(expressApp);
await this.initExpress(expressApp);
```
And then make the change to `initExpress` to accept a param

Reply:
I like this ^^^ the ordering of the routes is more explicit - also explain inline why it must be in this order

Reply:
Done!